mollie-payments-for-woocommerce.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Plugin Name: Mollie Payments for WooCommerce
+ * Plugin URI: https://www.mollie.com
+ * Description: Accept payments in WooCommerce with the official Mollie plugin
+ * Version: 5.1.8
```

Comment:
The version should be 5.2.1

Reply:
Here it says "5.1.8" most likely because that was the version from which I branched. If it still says that, then later changes weren't merged correctly into this branch.